ui: cluster overview node status color mismatch

vladlos opened the pull request:
We aggreagate rows for multiregion cluster in nodes list
table and used single method to calculate status badge for
aggregated and single node row. After adding additional
node status `NODE_STATUS_DRAINING` we have got
enum values overlap and that lead to `LIVE` badge with
`warning` color.
To fix that, and avoid same problem in future, changes for
splitting up aggregated and non aggregated color values
were made.

Resolves: #67274

Release note(ui); fix color mismatch of node status badge
on cluster overview page.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/68049)
<!-- Reviewable:end -->

vladlos committed:
ui: cluster overview node status color mismatch

We aggreagate rows for multiregion cluster in nodes list
table and used single method to calculate status badge for
aggregated and single node row. After adding additional
node status `NODE_STATUS_DRAINING` we have got
enum values overlap and that lead to `LIVE` badge with
`warning` color.
To fix that, and avoid same problem in future, changes for
splitting up aggregated and non aggregated color values
were made.

Resolves: #67274

Release note(ui); fix color mismatch of node status badge
on cluster overview page.

koorosh reviewed (approved):
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/68049)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @nathanstilwell and @Santamaura)



<!-- Sent from Reviewable.io -->

Santamaura reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/68049#-:-Mfcf3mf0cTaSK9Gd6Ud:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @nathanstilwell)



<!-- Sent from Reviewable.io -->

elkmaster commented:
bors r+
